pipelines/multi-branch/template.yml

```
@@ -0,0 +1,81 @@
+resource_types:
+- name: terraform
+  type: registry-image
+  source:
+    repository: ljfranklin/terraform-resource
+
+- name: gcs
+  type: registry-image
+  source:
+    repository: frodenas/gcs-resource
+
+resources:
+- name: branch
+  type: git
+  source:
+    uri: https://github.com/concourse/examples
+    branch: ((branch))
+
+- name: examples
+  type: git
+  source:
+    uri: https://github.com/concourse/examples
+
+- name: build-artifact
+  type: gcs
+  source:
+    bucket: concourse-examples
+    json_key: ((concourse_artifacts_json_key))
```

nit: keep it general, looks like this was copied from one of our pipelines
```suggestion
    json_key: ((artifacts_json_key))
```